### DISCUSSION TITLE
Gauge Intellisense

### DISCUSSION THREAD

**Discussion by AndyK28** [getgauge/gauge#2637] [General]
I am having an issue whereby my steps would be red for no reason stating that I have a duplicate step implementation even though it is the only step I have with that naming and it only has 1 implementation for it.
![Screenshot 2024-10-29 090248](https://github.com/user-attachments/assets/75ef33bb-832c-4c8b-98c9-917afe24aa9c)
![Screenshot 2024-10-29 090347](https://github.com/user-attachments/assets/ea08036b-e520-4d24-aeb4-4272d40702f2)

**Comment by jensakejohansson** [comment]
@AndyK28 What language are you using to implement your steps? C#?
